plugin/poi.vim

```diff
@@ -1,32 +1,56 @@
 " Highlight points of interest
-let s:match_base = ':match poi '
-au! VimEnter * execute ":autocmd InsertLeave * call <SID>MakeMatch()"
-au! ColorScheme * execute 'highlight poi ctermbg='.s:c_bg.' ctermfg='.s:c_fg.' guibg='s:g_bg.' guifg='.s:g_fg
-au! BufEnter * execute 'highlight poi ctermbg='.s:c_bg.' ctermfg='.s:c_fg.' guibg='s:g_bg.' guifg='.s:g_fg
-au! BufWinEnter * execute 'highlight poi ctermbg='.s:c_bg.' ctermfg='.s:c_fg.' guibg='s:g_bg.' guifg='.s:g_fg
-au! BufEnter * call <SID>MakeBuff()
-au! CursorHold * call <SID>MakeMatch()
-au! CursorMoved * call <SID>MakeMatch()
-au! CursorMovedI * call <SID>MakeMatch()
+let s:match_base = ':match poi1 '
```

Comment:
It appears you were aligning everything in your commits. For example look at lines in the "Highlight Lines" section of the README. Probably want to do the same here.